src/index/Index.cpp

```diff
@@ -290,16 +297,17 @@ void Index::createPermutationImpl(const string& fileName,
 
   LOG(INFO) << "Writing Meta data to file...\n";
```

Comment:
This log is now at the wrong place, right? After all we're done writing at this point